src/android/PurchasesPlugin.java

```diff
@@ -170,14 +169,14 @@ public void syncPurchases(CallbackContext callbackContext) {
         callbackContext.success();
     }
 
-    @PluginAction(thread = ExecutionThread.UI, actionName = "syncObserverModeAmazonPurchase")
-    public void syncObserverModeAmazonPurchase(String productID,
+    @PluginAction(thread = ExecutionThread.UI, actionName = "syncAmazonPurchase")
+    public void syncPurchase(String productID,
```

Comment:
Should this be `syncAmazonPurchase` instead of `syncPurchase`?

```suggestion
    public void syncAmazonPurchase(String productID,
```

Reply:
Ah right, I think it shouldn't matter, synce what matters is the naming above... But yeah, better to be consistent 👍